samples/springboot3/pet-store/pom.xml

```diff
@@ -43,7 +43,7 @@
         <dependency>
             <groupId>com.amazonaws.serverless</groupId>
             <artifactId>aws-serverless-java-container-springboot3</artifactId>
-            <version>[2.0-SNAPSHOT,)</version>
+            <version>2.0.0-M2</version>
```

Comment:
Let's still utilize a version range. `[2.0.0-SNAPSHOT,),[2.0.0-M1,)` should do the trick (related issue https://issues.apache.org/jira/browse/MNG-3092).